packages/terra-aggregate-translations/UpgradeGuide.md

```diff
@@ -1,6 +1,14 @@
 # Terra Aggregate Translations
 
-## Changes from version 3 to version 4
+## Changes from version 2 to version 3
+
+`terra-aggregate-translations v3` drops support for `react-intl 4` and below. Projects consuming `terra-aggregate-translations` version 3 should use `react-intl` version `>= 5`.
+
+`react-intl v5` no longer exports the `intlShape`. Consuming projects should replace the `intlShape` with `PropTypes.shape({ formatMessage: PropTypes.func })`. See [doc](https://formatjs.io/docs/intl#the-intl-object) for more information.
+
+Follow the upgrade guide of react-intl v5 to know more about the breaking changes : https://formatjs.io/docs/react-intl/upgrade-guide-5x 
```

Comment:
Most of the breaking changes happened in v3, I would link to the upgrade guides for v3 and v4 in addition to the v5 link so consumers can find all the breaking changes they need to be aware of for this jump.

Reply:
done : [cc89bf](https://github.com/cerner/terra-toolkit/pull/739/commits/cc89bf24134a5fded22ea5a8e135903729188072)